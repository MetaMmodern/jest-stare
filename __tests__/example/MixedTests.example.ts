describe("Mixed Passing and Failing Tests", () => {

    describe("Snapshot differences", () => {
        it("should fail with a snapshot difference", () => {
            const jsonObject: any = {
                name: "ein",
                breed: "corgi",
                height: 1,
                color: "brown and white"
            };
            expect(JSON.stringify(jsonObject, null, 2)).toMatchSnapshot();
        });
    });

    it("should fail due to an error", () => {
        throw new Error("This is an error failure");
    });

    it("Should fail due to a bad expectation", () => {
        expect(false).toEqual(true);
    })
});
