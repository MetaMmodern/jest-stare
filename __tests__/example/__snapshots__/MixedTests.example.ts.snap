// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mixed Passing and Failing Tests Snapshot differences should fail with a snapshot difference 1`] = `
"{
  \\"name\\": \\"ein\\",
  \\"breed\\": \\"corgi\\",
  \\"height\\": 1,
  \\"color\\": \\"brown and white\\"
}"
`;
